### PR TITLE
Metameta new recipe

### PR DESCRIPTION
**Pull request by pirovc**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

@bioconda/core

### CODE REVIEW

**Comment by daler** [comment]
@pirovc do you think it's worth adding a test to make sure snakemake can find all the installed configs and scripts?

**Comment by pirovc** [comment]
Some includes are dynamic and depend on user choice. Still I could add a dummy configuration file and trigger a dry-run with the default options. Would that suffice?

**Comment by daler** [comment]
I think that would improve the test; @johanneskoester any thoughts?

**Comment by johanneskoester** [comment]
Sounds good!

**Comment by pirovc** [comment]
@daler test added. If any file/include is missing or if snakemake is not running the test command is going to return an error.

File: recipes/metameta/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  skip: True # [osx]  
+  
+requirements:
+  run:
+    - snakemake >=3.9.0
+
+test:
+  commands:
+    - touch "metameta_dummy_file_test"
+    - metameta --configfile $PREFIX/opt/metameta/scripts/metameta_dummy_file_test.yaml -np > /dev/null
```
Comment (johanneskoester) [inline recipes/metameta/meta.yaml]: Please remove the piping into /dev/null (harder to debug if something goes wrong).

**Comment by daler** [comment]
Thanks @pirovc!

**Pull request merged**